### PR TITLE
python3Packages.ez_setup: init at 0.9.0

### PR DESCRIPTION
**Pull request by alex-ameen**
###### Motivation for this change

Required package for a project.

###### Things done

Added python package "ex_setup"

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/ez_setup/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{ stdenv
+, fetchPypi
+, buildPythonPackage
+}:
+buildPythonPackage rec {
```
Comment (matthiasbeyer) [inline pkgs/development/python-modules/ez_setup/default.nix]: Not sure whether you need `rec` here.

**Review by matthiasbeyer — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/ez_setup/default.nix (hunk 1 above)
Comment (BadDecisionsAlex) [reply pkgs/development/python-modules/ez_setup/default.nix]: Thanks for the feedback. I removed `rec`.

Comment (BadDecisionsAlex) [reply pkgs/development/python-modules/ez_setup/default.nix]: Hey for whatever reason the package breaks when `rec` is removed. I suspect that it has to do with the use of `inherit`?

**Comment by BadDecisionsAlex** [comment]
Merging multiple PRs.

**Pull request closed without merging**